spec/serverSpec.js

```diff
@@ -2,28 +2,52 @@
  * for these tests to pass. */
 
 const { expect } = require('chai');
-const request = require('request');
+const axios = require('axios');
 
 describe('the server', () => {
-  it('the data/documents are in the right format & adhere to the schema', (done) => {
-    request('http://localhost:3003/reviews/20', (error, response, body) => {
-      const result = JSON.parse(body);
-      const review = result[0];
-
-      expect(review).to.have.all.keys('_id', 'id', 'image_url', 'reviewer_name', 'star_rate', 'review_date', 'review_description', 'likes_count');
-      done();
-    });
+  let result;
+
+  // runs before all tests in this block
+  before((done) => {
+    axios('http://localhost:3003/reviews/20')
+      .then((response) => {
+        result = response.data;
+        done();
```

Comment:
Do you need to use "done()" with promises? Jest docs show using one or the other with async tests (https://jestjs.io/docs/en/asynchronous). Also, not sure if this would apply to your test, but this post suggests using "finally(done)" with promises instead of just "done()" (https://wietse.loves.engineering/testing-promises-with-mocha-90df8b7d2e35). I'm not very confident with promises, but I wanted flag this just in case.